src/integrationtests/java/com/aws/greengrass/integrationtests/util/ConfigPlatformResolver.java

```diff
@@ -76,7 +76,7 @@ public static void initKernelWithMultiPlatformConfig(Kernel kernel, URL path) th
         }
         if (!isWindows) {
             try {
-                String sysver = Exec.sh("uname -a").toLowerCase();
+                String sysver = Platform.getInstance().createNewProcessRunner().sh("uname -a").toLowerCase();
```

Comment:
Add a TODO for windows if there isn't one.

Reply:
Yes there is a `if (isWindows)` above this one. Good reminder for me to search whole codebase for "if (!isWindows)"!

Reply:
Yes I saw the if, I mean to add a TODO so we don't skip implementing this feature on windows

Reply:
Looking at the context now though, it doesn't matter in this case.